src/modules/App.js

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable */
+import { getData, countOccurrences } from './ApiSimpson.js';
+import iconLike from '../assests/like.png';
+import getLikes from './GetLikes.js';
+import sendLike from './SendLike.js';
+import addComment from './addComments.js';
+import { API_URL1 } from './Vars.js';
+import getComments from './GetComments.js';
+
+// Render cards
+async function renderCards() {
+  // Data from simpson api
+  const data = await getData(API_URL1);
+  const CardsContainer = document.getElementById('dinamic-content');
+
```

Comment:
- [ ] Please, It's one of the project requirements you use ES6, let's use `arrow` syntax for functions instead of JavaScript Function Expressions because in this project you learn how to use ES6 new features. Kindly make this :wink: 💯 

- For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)

Reply:
well resolved! ✔️